src/rastervision/tagging/tasks/predict.py

```diff
@@ -87,7 +88,27 @@ def compute_probs(run_path, model, options, generator, split):
         normalize=True, only_xy=False)
 
     for batch_ind, batch in enumerate(split_gen):
-        y_probs.append(model.predict(batch.x))
+        # Performing safe augmentations on images one by one, predicting
+        # probs and taking average, if calculating test probabilities
+        if split == TEST:
+            for img_ind in range(batch.x.shape[0]):
+                img = batch.x[img_ind, :, :, :]
+                rot_batch = []
+                for rotation in range(4):
+                    rot_img = np.rot90(img, rotation)
+                    rot_img = np.expand_dims(rot_img, axis=0)
+                    rot_batch.append(rot_img)
+                rot_batch.append(np.flipud(rot_img))
```

Comment:
It turns out that `rot_img` is the same as `img` after rotating it 4 times, but I think it would be clearer to call `flipud` on `img`. Same for the next line.